app/models/EmailTemplate.scala

```diff
@@ -172,8 +174,10 @@ object EmailTemplate {
       periodEndMonth <- metadata.get("PeriodEndMonth")
       periodEndYear  <- metadata.get("PeriodEndYear")
       dutyText        = statementType match {
-                          case "Excise" => "The total excise owed will be collected by direct debit on or before"
-                          case _        => "The total Duty and VAT owed will be collected by direct debit on or after"
+                          case EXCISE_STT_TYPE | DD1920_STT_TYPE =>
```

Comment:
Better to move this into a private method with meaningful name to make it more readable